db/moonstreamdb/models.py

```diff
@@ -147,7 +147,13 @@ class EthereumLabel(Base):  # type: ignore
     address_id = Column(
         Integer,
         ForeignKey("ethereum_addresses.id", ondelete="CASCADE"),
-        nullable=False,
+        nullable=True,
+        index=True,
+    )
+    transaction_hash = Column(
+        VARCHAR(256),
+        ForeignKey("ethereum_transactions.hash", ondelete="CASCADE"),
+        nullable=True,
         index=True,
     )
```

Comment:
My proposal to add:
```python
    @hybrid_property
    def target_id(self):
        if self.address_id is not None:
            return self.address_id
        elif self.transaction_hash is not None:
            return self.transaction_hash
        else:
            return None
 ```
 to avaoid empty `address_id` with empty `transaction_hash`

Reply:
For some labels, the target id is actually a pair of `(transaction_hash, address)`. Do `hybrid_property`s support that?